### DISCUSSION TITLE
Output parameter for commands

### DISCUSSION THREAD

**Discussion by expiscornovus** [microsoft/powerplatform-build-tools#704] [Ideas]
I see some commands have --json parameter, but not all of them. I would find it useful to get a generic output parameter for all the commands, just like in the CLI for Microsoft 365 module.

-o, --output [output]

Output types possibly: json, text, csv, md, none. Default json.

Btw, out of interest. What is the current format that is outputted, is it tab delimited?

**Comment by simon-sabin-BE** [comment]
duplicate of #463
